ocs_ci/utility/utils.py

```diff
@@ -1796,3 +1797,28 @@ def skipif_ocs_version(expressions):
         skip_this = skip_this and eval(comparision_str)
     # skip_this will be either True or False after eval
     return skip_this
+
+
+def get_ocs_version_from_tag(tag):
+    """
+    Parse major.minor version from OCS image tag.
+
+    Args:
+        tag (str): version tag from the image
+
+    Returns
+        str: Version in x.y format
+
+    Raises:
+        ValueError: In case tag which we cannot parse passed.
+
+    """
+    try:
+        version = tag.split(':')[1].lstrip("latest-")
+        version = Version.coerce(version)
```

Comment:
why coerce? should work without it right?

Reply:
See the previous discussions. this allows us to support `latest-4.Y` as well as `4.2.Z-<release>` tags.

Reply:
But the 'latest-' is already stripped in line 1817 leaving with only supported version format?

Reply:
@vasukulkarni '4.2' is invalid string in Version. To overcome that either we have to use coerce or pass "partial=True" as parameter to Version like 
``` Version('4.2', partial=True)```



https://pypi.org/project/semantic-version/

Reply:
I see, thanks!